README.md

```diff
@@ -1,42 +1,71 @@
-# BTCPay plugin for GrandNode - accept Bitcoin payments
+# BTCPay Server – Accept Bitcoin payments in GrandNode
 
+![BTCPay GrandNode Banner](GrandNodeAcceptBitcoin.png)
 
 ## Plugin Overview
 
-This plugin allows you to easily integrate Bitcoin payments into your GrandNode website using BTCPay Server — a free, self-hosted and open-source payment gateway solution designed to revolutionize Bitcoin payments. Our seamless integration with GrandNode ensures a hassle-free connection to your self-hosted BTCPay Server. 
+This plugin allows you to easily start accepting Bitcoin payments in your GrandNode e-commerce store using BTCPay Server — a free, self-hosted and open-source payment gateway. 
 
-Experience the simplicity of accepting Bitcoin payments with just a few straightforward steps. You can configure the plugin either automatically or manually, depending on your preferences and requirements.
+## Features
 
-## Automatic Configuration
+- 0% Fees: Enjoy a payment gateway with no fees. Yes, really!
+- No complicated KYC needed, you're your own, self-hosted payment gateway
+- Automated invoice management and refunds
+- Simple and customizable checkout UX, available in 30+ languages
+- Direct payments to your wallet, no intemediary
+- Reporting and accounting – CSV exports
+- Real-time exchange price tracking for correct payment amounts
+- Community-driven support
+- Extensive documentation and video tutorials
+
+## Getting Started 
+
+### Download and installation
+
+The plugins can be downloaded through GrandNode Extension Directory or GitHub. 
+
+For installing the plugin on GrandNode, you can either upload the plugin directly to the `/plugins` folder in your GrandNode main directory and restart your application, or use the Admin Panel to upload the plugin. 
+
+In the **Admin Panel**, navigate to **Plugins -> Local Plugins** and click the **"Upload"** button. Ensure the plugin ZIP file has the correct structure for successful upload. After uploading, find the plugin in the list, click "Install" next to it, and enable it if needed​.
```

Comment:
"Ensure the plugin ZIP file has the correct structure for successful upload." not sure if this bothers users, we need to make sure the structure is right when generating the zip?

Reply:
Done